dao/src/main/resources/sql/schema-entities.sql

```diff
@@ -64,6 +64,12 @@ CREATE TABLE IF NOT EXISTS alarm (
     cleared boolean
 );
 
+CREATE TABLE IF NOT EXISTS alarm_types (
+    tenant_id uuid NOT NULL,
+    type varchar(255) NOT NULL,
+    CONSTRAINT tenant_id_type_unq_key UNIQUE (tenant_id, type)
```

Comment:
CONSTRAINT fk_entity_alarm_id FOREIGN KEY (alarm_id) REFERENCES alarm(id) ON DELETE CASCADE

(for tenant)